### PR TITLE
Sqlitecpp to mainrepo

### PR DESCRIPTION
**Pull request by JakubFornadel**

### CODE REVIEW

File: libraries/SQLiteCpp/.cproject [hunk 1]
```diff
@@ -0,0 +1,538 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?fileVersion 4.0.0?><cproject storage_type_id="org.eclipse.cdt.core.XmlProjectDescriptionStorage">
```
Comment (MatusKysel) [inline libraries/SQLiteCpp/.cproject]: Please remove eclipse project files

File: libraries/SQLiteCpp/.gitbugtraq [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# .gitbugtraq for Git GUIs (SmartGit/TortoiseGit) to show links to the Github issue tracker.
```
Comment (MatusKysel) [inline libraries/SQLiteCpp/.gitbugtraq]: Also this one

File: libraries/SQLiteCpp/.project [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (MatusKysel) [inline libraries/SQLiteCpp/.project]: Also this one

File: libraries/SQLiteCpp/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,183 @@
+# Copyright (c) 2012-2018 Sebastien Rombauts (sebastien.rombauts@gmail.com)
```
Comment (MatusKysel) [inline libraries/SQLiteCpp/.travis.yml]: Also this one

File: libraries/SQLiteCpp/_config.yml [hunk 5]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```
Comment (MatusKysel) [inline libraries/SQLiteCpp/_config.yml]: also delete this one

File: libraries/SQLiteCpp/appveyor.yml [hunk 6]
```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2012-2018 Sebastien Rombauts (sebastien.rombauts@gmail.com)
+
+# build format
```
Comment (MatusKysel) [inline libraries/SQLiteCpp/appveyor.yml]: also delete this one

File: libraries/SQLiteCpp/build.bat [hunk 7]
```diff
@@ -0,0 +1,25 @@
+@REM Copyright (c) 2012-2018 Sebastien Rombauts (sebastien.rombauts@gmail.com)
```
Comment (MatusKysel) [inline libraries/SQLiteCpp/build.bat]: also delete this one

File: libraries/SQLiteCpp/build.sh [hunk 8]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```
Comment (MatusKysel) [inline libraries/SQLiteCpp/build.sh]: also delete this one

File: libraries/SQLiteCpp/cpplint.py [hunk 9]
```diff
@@ -0,0 +1,4807 @@
+#!/usr/bin/python
```
Comment (MatusKysel) [inline libraries/SQLiteCpp/cpplint.py]: Also remove, we are not using Google style

**Comment by MatusKysel** [comment]
Also please remove icon

**Review by MatusKysel — APPROVED** [review]

**Review by ejossev — APPROVED** [review]

**Pull request merged**